deps.ts

```diff
@@ -1,12 +1,14 @@
 export {
   CXEvalResultKind,
   CXVisitorResult,
-} from "https://deno.land/x/libclang@1.0.0-beta.4/include/typeDefinitions.ts";
+} from "https://deno.land/x/libclang@1.0.0-beta.8/include/typeDefinitions.ts";
 export {
   CXIndex,
   CXFile,
   CXCursor,
   CXCursorKind,
+  CXObjCPropertyAttrKind,
   type CXType,
-} from "https://deno.land/x/libclang@1.0.0-beta.4/mod.ts";
-export { CXChildVisitResult } from "https://deno.land/x/libclang@1.0.0-beta.4/include/typeDefinitions.ts";
+  type AvailabilityEntry,
+} from "https://deno.land/x/libclang@1.0.0-beta.8/mod.ts";
+export { CXChildVisitResult } from "https://deno.land/x/libclang@1.0.0-beta.8/include/typeDefinitions.ts";
```

Comment:
I think this should also be available from the `mod.ts`. At least my intention has been that `mod.ts` would export everything needed.